pkgs/development/python-modules/datalad/default.nix

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
Also you need to rewrite this after https://github.com/NixOS/nixpkgs/pull/329398 merge

Reply:
done